src/WebJobs.Script/Host/ProxyMetadataManager.cs

```diff
@@ -119,7 +119,7 @@ private ProxyMetadataInfo LoadFunctionMetadata()
                     proxyMetadata.Bindings.Add(bindingMetadata);
 
                     proxyMetadata.Name = proxyName;
-                    proxyMetadata.ScriptType = ScriptType.Unknown;
+                    proxyMetadata.Language = "Unknown";
```

Comment:
How is this "Unknown" value used? I wonder if rather than setting it, we just leave it null, which can then signal unknown?

Reply:
done!